charts/atlantis/values.yaml

```diff
@@ -37,6 +37,14 @@ orgWhitelist: <replace-me>
 #   baseURL: https://bitbucket.yourorganization.com
 # (The chart will perform the base64 encoding for you for values that are stored in secrets.)
 
+# If using Azure DevOps, specify like the following:
+# azuredevops:
```

Comment:
it's better to have a flag like `enabled: false`

Reply:
yeah that's a better pattern but the other VCS stanzas don't have that so I think this is more consistent.